onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -675,7 +675,8 @@
     <string name="preferences_save_title">Save to storage</string>
     <string name="preferences_save_summary">Saves your starred and recent stops and routes</string>
     <string name="preferences_restore_title">Restore from storage</string>
-    <string name="preferences_restore_summary">Restores from previous save. After selecting this option you\'ll be asked to pick a file. Please select \"db.backup\" or \"OneBusAway.backup\", which will be in a folder \"OBABackups\", likely on your SD card or under your \"Documents\" folder.</string>
+    <string name="preferences_restore_summary">Restores from a previous backup. After selecting this option, you\'ll be asked to pick a file. Please select the backup file (e.g., "db.backup" or "OneBusAway.backup"), which may be in your "Documents" folder or any location where you saved the backup.
```

Comment:
please be sure to update the other localized variants of `preferences_restore_summary`!

Reply:
yes, I checked them... they only contain `Restore from the previous backup`.